public/static/docs/command-reference/import.md

```diff
@@ -15,17 +15,17 @@ import.
 usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to data within DVC repository.
+  url         URL of DVC project or Git repository to download from.
+  path        Path to data within the repository.
 ```
 
 ## Description
 
-DVC provides an easy way to reuse datasets, intermediate results, ML models, or
-other files and directories tracked in another <abbr>DVC repository</abbr> into
-the workspace. The `dvc import` command downloads such a <abbr>data
-artifact</abbr> in a way that it is tracked with DVC, so it can be updated when
-the data source changes. (See `dvc update`.)
+Provides an easy way to reuse datasets, intermediate results, ML models, or
+other files and directories tracked in another <abbr>DVC project</abbr> or Git
+repository into the workspace. The `dvc import` command downloads such a
+<abbr>data artifact</abbr> in a way that it is tracked with DVC, so it can be
```

Comment:
```suggestion
file or <abbr>artifact</abbr> in a way that it is tracked with DVC, so it can be
```

Reply:
Because the tooltip that comes up for "data artifact" says:

```
Any data file or directory, as well as intermediate or final result (such as
extracted features or a ML model file) that is under DVC control. Refer to
[Versioning Data and Model Files]
(/doc/use-cases/versioning-data-and-model-files) for more details. A.k.a
**output*.
```

and we're not only talking about stuff under DVC control.

Reply:
I tried fixing this myself but I don't have permission to push to your branch. For future ref: feel free to allow maintainers of the repo to push to your PRs, it may save some time for small things like this 🙂

Reply:
I tried to change this 'allow edits from maintainers' checkbox was already checked; I don't know what the problem was

Reply:
Just noticed this. Thanks no worries it seems actually there's not many changes left in the PR after merging with `master` anyway. Thanks though!